portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
```

Comment:
This file isn't a servlet, so it should be under a different folder, e.g. data.

Reply:
Fixed!